tests/PEARX/ChannelTest.php

```diff
@@ -64,6 +64,7 @@ function getChannels()
     }
 
     /**
+     * @group noPearSymfony
```

Comment:
It seems that the `pear.symfony.com` host is dead. Adding the `@group` annotation to ignore these tests.